packages/yourloops/pages/hcp/patients/list.tsx

```diff
@@ -0,0 +1,206 @@
+import { errorTextFromException, setPageTitle } from "../../../lib/utils";
+import { TeamContext, useTeam } from "../../../lib/team";
+import PatientsTable from "./table";
+import PatientsCards from "./cards";
+import { Patient } from "../../../models/patient";
+import { PatientListProps } from "./models";
+import { comparePatients } from "./utils";
+
+const log = bows("PatientListPage");
+
+// eslint-disable-next-line no-magic-numbers
+const throttleSearchMetrics = _.throttle(metrics.send, 10000, { trailing: true });
+
+function PatientList(props: PatientListProps): JSX.Element {
+  const { filter, filterType } = props;
+  const historyHook = useHistory();
+  const { t } = useTranslation("yourloops");
+  const theme = useTheme();
+  const matchesMediaSizeSmall = useMediaQuery(theme.breakpoints.down("sm"));
+  const authHook = useAuth();
+  const teamHook = useTeam();
+  const [loading, setLoading] = React.useState<boolean>(true);
+  const [errorMessage, setErrorMessage] = React.useState<string | null>(null);
+  const [order, setOrder] = React.useState<SortDirection>(SortDirection.asc);
+  const [orderBy, setOrderBy] = React.useState<PatientTableSortFields>(PatientTableSortFields.patientFullName);
+  const flagged = authHook.getFlagPatients();
+
+  const updatePatientList = (
+    teamHook: TeamContext,
+    flagged: string[],
+    filter: string,
+    filterType: FilterType | string,
+    orderBy: PatientTableSortFields,
+    order: SortDirection
+  ) => {
+    let filteredPatients = teamHook.filterPatients(filterType, filter, flagged);
+    filteredPatients = teamHook.computeFlaggedPatients(filteredPatients, flagged);
+    // Sort the patients
+    filteredPatients.sort((a: Patient, b: Patient): number => {
+      const c = comparePatients(a, b, orderBy);
+      return order === SortDirection.asc ? c : -c;
+    });
+    const searchByName = filter.length > 0;
+    if (searchByName) {
+      throttleSearchMetrics("trackSiteSearch", "patient_name", "hcp", filteredPatients.length);
+    }
+    return filteredPatients;
+  };
+
+  const handleRefresh = async (force = false) => {
+    log.debug("handleRefresh:", { force });
+    setLoading(true);
+    setErrorMessage(null);
+    try {
+      await teamHook.refresh(force);
+    } catch (reason: unknown) {
+      log.error("handleRefresh", reason);
+      const errorMessage = t("error-failed-display-teams", { errorMessage: errorTextFromException(reason) });
+      setErrorMessage(errorMessage);
+    }
+    setLoading(false);
+  };
+
+  const handleSelectPatient = (patient: Patient): void => {
+    metrics.send("patient_selection", "select_patient", flagged.includes(patient.userid) ? "flagged" : "not_flagged");
+    historyHook.push(`/patient/${patient.userid}`);
+  };
+
+  const handleFlagPatient = async (userId: string): Promise<void> => {
+    metrics.send("patient_selection", "flag_patient", flagged.includes(userId) ? "flagged" : "un-flagged");
+    await authHook.flagPatient(userId);
+  };
```

Comment:
It can be done when the cards component is deleted (and depending if we have another component to handle smaller screens, otherwise it will lead to code duplication)